packages/api/src/api.ts

```diff
@@ -85,12 +85,20 @@ export function createApiRouter(localPath: string): express.Router {
     )
   })
 
-  router.get('/messages/:messageId/audio', (req, res) => {
+  router.get('/messages/:messageId/audio', async (req, res) => {
     // Special MessagePaylod that is not json, only bin'd file.
-    res.status(200).end()
+    const audio = await store.getAudio(req.params.messageId)
+    if (!audio) {
+      res.status(404).end()
+    }
+    res.status(200).json(
+      makeResponse<Api.AudioPayload>({audio: {data: audio || ''}}),
```

Comment:
@patrickhulce what format will Twilio want?

I doubt `{payload: {audio: {data: 010101010101001}}}` is what they want?

Reply:
We could just use express sendFile or something?

Reply:
yeah can just `.send` the buffer itself no need to do JSON

Reply:
I could also see us return base64 in JSON if the requested content type is JSON?